package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@parity/ui",
   "description": "Parity React UI components",
-  "version": "3.1.9",
+  "version": "3.1.10",
```

Comment:
CI will automatically bump version. This will release the next one as 3.1.11, so not a big deal, but can be removed.

Reply:
Reverted!